core/src/components/header/test/condense/header.e2e.ts

```diff
@@ -7,24 +7,37 @@ test.describe('header: condense', () => {
     skip.rtl();
 
     await page.goto('/src/components/header/test/condense');
-    const header = page.locator('#collapsibleHeader');
+    const largeTitleHeader = page.locator('#largeTitleHeader');
+    const smallTitleHeader = page.locator('#smallTitleHeader');
     const content = page.locator('ion-content');
 
-    await expect(header).toHaveAttribute('aria-hidden', 'true');
+    await expect(smallTitleHeader).toHaveAttribute('aria-hidden', 'true');
+
+    expect(await largeTitleHeader.screenshot()).toMatchSnapshot(
+      `header-condense-large-title-initial-diff-${page.getSnapshotSettings()}.png`
+    );
 
     await content.evaluate((el: HTMLIonContentElement) => el.scrollToBottom());
     await page.waitForChanges();
 
+    expect(await smallTitleHeader.screenshot()).toMatchSnapshot(
+      `header-condense-large-title-collapsed-diff-${page.getSnapshotSettings()}.png`
+    );
+
     /**
      * Playwright can't do .not.toHaveAttribute() because a value is expected,
      * and toHaveAttribute can't accept a value of type null.
      */
-    const ariaHidden = await header.getAttribute('aria-hidden');
+    const ariaHidden = await smallTitleHeader.getAttribute('aria-hidden');
     expect(ariaHidden).toBeNull();
 
     await content.evaluate((el: HTMLIonContentElement) => el.scrollToTop());
     await page.waitForChanges();
 
-    await expect(header).toHaveAttribute('aria-hidden', 'true');
+    expect(await largeTitleHeader.screenshot()).toMatchSnapshot(
+      `header-condense-large-title-final-diff-${page.getSnapshotSettings()}.png`
+    );
```

Comment:
I'm not sure this screenshot is necessary? It's going to be identical to the initial state, so verifying `aria-hidden` seems sufficient to me.